deploy/crds/spinnaker.io_spinnakerservices_crd.yaml

```diff
@@ -59,6 +59,15 @@ spec:
         spec:
           description: SpinnakerServiceSpec defines the desired state of SpinnakerService
           properties:
+            accounts:
+              properties:
+                dynamic:
```

Comment:
enabled -> use CRDs. dynamic -> inject at runtime

Reply:
If both these settings only apply to accounts defined in CRD, maybe we can rename this field to `crdAccounts`, otherwise it could cause confusion to have `accounts.enabled: false` but still have working all accounts defined in main config file/profile files.

Reply:
I'd like to avoid calling anything `crd` as it's not even the right term. `SpinnakerAccounts` may be better?

Reply:
Better!